README.md

```diff
@@ -163,7 +165,7 @@ class MainActivity: FlutterFragmentActivity() {}
 
 ## API
 
-### `configure(String appKey)`
+### `configure(RowndConfigure({ appKey: String }))`
```

Comment:
Just a nit, but is it possible to call this `RowndConfig` instead of `RowndConfigure`?

Reply:
Yes nice add! Updated